hydradx/tests/test_omnipool_amm.py

```diff
@@ -1017,7 +1017,7 @@ def test_sell_stableswap_for_stableswap(initial_state: oamm.OmnipoolState):
 def test_slip_fees(initial_state: oamm.OmnipoolState, lrna_slip_rate: float, asset_slip_rate: float):
     initial_state.lrna_fee = oamm.slip_fee(lrna_slip_rate, minimum_fee=0.0001)
     initial_state.asset_fee = oamm.slip_fee(asset_slip_rate, minimum_fee=0.0001)
-    initial_agent = Agent(holdings={tkn: 1000000 for tkn in initial_state.asset_list})
+    initial_agent = Agent(holdings={tkn: 10000000 for tkn in initial_state.asset_list})
```

Comment:
Was this necessary for some reason? Seems like a weird fix... either should work I'd think?

Reply:
I think it did at some point seem to fix something, but I also don't see why that would work or be necessary. However, it also seems fine.